newsletter_automation/newsletter/routes.py

```diff
@@ -21,6 +22,8 @@
 from prometheus_client import generate_latest, CONTENT_TYPE_LATEST
 from prometheus_client import Counter, Gauge
 import psutil
+import time
```

Comment:
Do we need these imports?

Reply:
Removed the time and wraps import. The rest is necessary